Formula/cmake-docs.rb

```diff
@@ -1,10 +1,9 @@
 class CmakeDocs < Formula
   desc "Documentation for CMake"
   homepage "https://www.cmake.org/"
-  url "https://github.com/Kitware/CMake/releases/download/v3.23.0/cmake-3.23.0.tar.gz"
-  mirror "http://fresh-center.net/linux/misc/cmake-3.23.0.tar.gz"
-  mirror "http://fresh-center.net/linux/misc/legacy/cmake-3.23.0.tar.gz"
```

Comment:
The first mirror moves to the second mirror when a new version is released.

Reply:
Good point, I'll restore it.

`glew` linkage failure is unrelated.

BTW why test reinstalls all those formulas without building them from source? IMHO, it's just a waste of time.

Reply:
It only tests things which are defined with `depends_on "cmake" => :test` (or at runtime, which there is none).

It also works recursively though and I agree that shouldn't be the case here. I'll add a label for that.